misaligned.c

```diff
@@ -1,21 +1,50 @@
 #include <stdio.h>
 #include <assert.h>
 
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+typedef struct 
+{
+    int pairID;
+    char* majorColor;

```

Comment:
To handle the compiler error, you can declare these two (`majorColor` and `minorColor`) as `const char*`
The `const` here means the contents of the pointer (string) cannot change. But the pointer itself can be assigned later.